src/polyvec/polyvec.rs

```diff
@@ -1,22 +1,33 @@
-use crate::polyvec::ff::FiniteRing;
-use crate::polyvec::vector::{Dot, Vector};
-use std::ops::{Index, IndexMut};
+use crate::polyvec::structures::{FiniteRing, RingModule};
 
 #[derive(Clone)]
-pub struct Polyvec<T: FiniteRing> {
-    coefficients: Vec<T>,
-    dimension: usize,
+pub struct PolyVec<T: FiniteRing> {
+    pub coefficients: Vec<T>,
+    pub dimension: usize,
```

Comment:
why pub since we have getters and setters ?

Reply:
There is no secret data so setters/getters in this case seem overkill, plus exposing `coefficients` provides iterators and item which makes it more practical to use